lib/transaction_repo.rb

```diff
@@ -22,4 +22,55 @@ def transaction_instances(csv_files)
   def all
     @transaction_list
   end
+
```

Comment:
These methods look great, I think we can include a new method to lower run time.